app/models/Operator.scala

```diff
@@ -23,8 +23,8 @@ case class Operator(
   name: Option[String]         = None,
   email: Option[String]        = None,
   role: Role                   = Role.CLASSIFICATION_OFFICER,
-  memberOfTeams: List[String]  = List.empty,
-  managerOfTeams: List[String] = List.empty,
+  memberOfTeams: Seq[String]   = Seq.empty,
+  managerOfTeams: Seq[String]  = Seq.empty, //being a member is the same as being the manager, don't use this
```

Comment:
Shall we remove this from the backend and frontend in a follow-up PR?

Reply:
yes, that would be good